pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+DJANGO_SETTINGS_MODULE = Xpense.settings
+django_debug_mode = true
+python_files = tests.py test_*.py *_tests.py
```

Comment:
A newline at end of the file is missing

Reply:
#35

Reply:
@adiabramovitch @shlior7 was this resolved?